src/main/webapp/i18n/de/complaint.json

```diff
@@ -11,6 +11,7 @@
             "descriptionExtended": "Alle Studierenden dürfen jeweils insgesamt bis zu {{ maxComplaintNumber }} Beschwerden in diesem Kurs einreichen. Du hast noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
             "descriptionTeam": "Ihr könnt insgesamt bis zu {{ maxComplaintNumber }} Beschwerden als Team in diesem Kurs einreichen.",
             "descriptionTeamExtended": "Jedes Team darf insgesamt bis zu {{ maxComplaintNumber }} Beschwerden in diesem Kurs einreichen. Ihr habt noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
+            "exclusivityDisclaimer": "Beschwerden und Anfragen für weiteres Feedback sind exklusiv zueinander. Du kannst nur entweder die eine oder die andere einreichen.",
```

Comment:
Maybe a bit better to understand: ".... Du kannst entweder die Beschwerde oder die Anfrage einreichen."